scripts/g.extension/g.extension.py

```diff
@@ -1940,10 +1940,11 @@ def main():
         list_installed_extensions(toolboxes=flags['t'])
         return 0
 
-    if flags['d']:
+    if flags['d'] or flags['i']:
+        flag = 'd' if flags['d'] else 'i'
         if options['operation'] != 'add':
-            grass.warning(_("Flag 'd' is relevant only to"
-                            " 'operation=add'. Ignoring this flag."))
+            grass.warning(_("Flag '%s' is relevant only to"
+                            " 'operation=add'. Ignoring this flag.") % flag)
```

Comment:
@tmszi Please use rather `format()` function (which is already used in `g.extension` code).

Reply:
All right, I changed it.